resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/CircuitBreakerMetrics.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *
+ *  Copyright 2017: Robert Winkler
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+package io.github.resilience4j.metrics;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.MetricSet;
+
+import java.util.Map;
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import io.github.resilience4j.circuitbreaker.CircuitBreakerRegistry;
+import javaslang.collection.Array;
+import javaslang.collection.Seq;
+
+import static com.codahale.metrics.MetricRegistry.name;
+
+/**
+ * An adapter which exports {@link CircuitBreaker.Metrics} as Dropwizard Metrics Gauges.
+ */
+public class CircuitBreakerMetrics implements MetricSet{
+
+    private final MetricRegistry metricRegistry = new MetricRegistry();
+
+    private CircuitBreakerMetrics(Seq<CircuitBreaker> circuitBreakers){
+        circuitBreakers.forEach(circuitBreaker -> {
+            metricRegistry.register(name("resilience4j.circuitbreaker", circuitBreaker.getName(), "successful"),
```

Comment:
You can move `circuitBreaker.getMetrics()` and `circuitBreaker.getName()` to separate variables on the start of the lambda.